src/Command/ConsumerCommand.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Skrz\Bundle\BunnyBundle\Command;
+
+use Bunny\Channel;
+use Bunny\Client;
+use Bunny\Message;
+use Bunny\Protocol\MethodBasicQosOkFrame;
+use Bunny\Protocol\MethodQueueBindOkFrame;
+use Bunny\Protocol\MethodQueueDeclareOkFrame;
+use InvalidArgumentException;
+use Skrz\Bundle\BunnyBundle\Exception\BunnyException;
+use Skrz\Bundle\BunnyBundle\Queue\BunnyConsumerInterface;
+use Skrz\Bundle\BunnyBundle\Queue\BunnyTickingConsumer;
+use Skrz\Bundle\BunnyBundle\Queue\BunnyTickingConsumerInterface;
+use Skrz\Bundle\BunnyBundle\Service\BunnyManager;
+use Skrz\Bundle\BunnyBundle\Service\ContentTypes;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Serializer\SerializerInterface;
+
+/** @author Lukas Senfeld <skrz@senfeld.net> */
+class ConsumerCommand extends Command
+{
+
+	/** @var BunnyConsumerInterface[] */
+	private iterable $consumers;
+	/**
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.PropertyTypeHint.MissingNativeTypeHint
+	 * @var string|null The default command name
+	 */
+	protected static $defaultName = 'bunny:consumer';
```

Comment:
proč je tady typ pomocí anotace?

Reply:
prosím sjednotit uvozovky VS apostrofy, koukám že někde u konstant (+ i v kodu) máš `'` a jinak používáš `"` -> jsem pro `"`

Reply:
obecne jsem hlavne pro `'` teda pokud vidis nekde uvozovy kde nejsou treba tak to pojdme zmenit

Reply:
@senfix  ok, šlo mi hlavně o sjednocení, díky

Jinak za mne ideál phpcs do budoucna, který to ohlídá